### PR TITLE
extract host url to consts

### PR DESCRIPTION
**Pull request by igracia**
Extracts fixed proto and hostname from host url

### CODE REVIEW

**Comment by igracia** [comment]
@YaraGomezSaiz @miriamela can you please review?

File: webpack.config.js [hunk 1]
```diff
@@ -5,7 +5,10 @@ var PrettierPlugin = require("prettier-webpack-plugin");
 const ErrorOverlayPlugin = require('error-overlay-webpack-plugin');
 
 const port = 3000;
-let publicUrl = `http://localhost:${port}`;
+const schema = 'http'
```
Comment (YaraGomezSaiz) [inline webpack.config.js]: @igracia is possible to change to "https" instead "http"?

**Comment by miriamela** [comment]
Hi! here I am

File: webpack.config.js (hunk 1 above)
Comment (miriamela) [inline webpack.config.js]: this should be different

**Pull request closed without merging**